src/applications/vaos/appointment-list/pages/RequestedAppointmentDetailsPage/RequestedAppointmentDetailsPage.jsx

```diff
@@ -52,11 +51,7 @@ export default function RequestedAppointmentDetailsPage() {
   useEffect(
     () => {
       if (appointment) {
-        let title = `${isCanceled ? 'Canceled ' : 'Pending '}${
-          isCC ? 'Community care' : 'VA'
-        } ${typeOfCareText} appointment`;
```

Comment:
This is the only place where typeOfCareText was used but it was immediately overwritten by the next assignment below.